### PR TITLE
Added own implementation for paste-from-word plugin

### PR DESCRIPTION
**Pull request by wachterjohannes**
This PR add a very simple but complete implementation for the paste-from-word plugin. We want to avoid any formation which was introduced in word.

### CODE REVIEW

File: husky_components/ckeditor/plugins/paste-from-word/main.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                                    {
+                                        type: 'cancel'
+                                    },
+                                    {
+                                        type: 'ok',
+                                        text: 'OK'
+                                    }
+                                ],
+                                okCallback: function() {
+                                    var text = this.$el.find('textarea').val();
```
Comment (danrot) [inline husky_components/ckeditor/plugins/paste-from-word/main.js]: Is the fact that we use a `textarea` removing all the stuff we don't want? Or where else is that happening?

**Comment by chirimoya** [comment]
@wachterjohannes ping

**Comment by wachterjohannes** [comment]
@chirimoya @danrot this was a test to make the PasteFromWord a bit more "strict" i will rebase it and maybe you can test it and if you think its good we can merge.

File: husky_components/ckeditor/plugins/paste-from-word/main.js (hunk 1 above)
Comment (wachterjohannes) [inline husky_components/ckeditor/plugins/paste-from-word/main.js]: yes we use the testarea (: which works for quite good for me.

**Comment by danrot** [comment]
Do I have to have a special word file for testing that? Or what exactly does the old solution have a problem with?

**Comment by wachterjohannes** [comment]
maybe @berndhep has one

**Comment by berndhep** [comment]
@wachterjohannes Where can I test it?

**Comment by wachterjohannes** [comment]
@danrot can we deploy it somewhere? or should we test it locally on my machine?

**Comment by danrot** [comment]
@wachterjohannes Can you test it on your machine? Less effort for me 😄

**Comment by danrot** [comment]
The buttons in the overlay are not correct.

**Comment by wachterjohannes** [comment]
@danrot fixed

**Pull request merged**